packages/manager/modules/pci/src/projects/project/data-processing/home/data-processing-home.html

```diff
@@ -69,7 +69,7 @@
                         <oui-tile-actions>
                             <button
                                 data-placement="end"
-                                data-ng-click=""
+                                data-ng-click="$ctrl.onAddNotebookClick()"
                                 class="oui-button oui-button_primary oui-button_icon-left"
```

Comment:
If you use the "oui-button" component, you can delete this line and use attributes like variant etc.

Reply:
Unfortunately, I tried it and that brings conflict with data-track directives that are also asking for a new scope. I could do the tracking in functions but that would add complexity so I'm not sure it would be worth it ... What do you think ?

Reply:
Okay, it's fine for me.